src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -1,35 +1,52 @@
 ---
-title: Debugging in VS Code
+title: Depurando en VS Code
 type: cookbook
 order: 8
 ---
 
-Every application reaches a point where it's necessary to understand failures, small to large. In this recipe, we explore a few workflows for VS Code users who would like to debug their application in the browser.
+Toda aplicación alcanza un punto donde es necesario entender los errores, desde
+los más chicos a los más grandes. En esta receta, exploraremos algunos flujos de
+trabajo para usuarios de VS Code a quienes les gustaría depurar su aplicación
+en el navegador.
 
-This recipe shows how to to debug [Vue CLI](https://github.com/vuejs/vue-cli) applications in VS Code as they run in the browser.
+Esta receta muestra como depurar aplicaciones [Vue CLI](https://github.com/vuejs/vue-cli)
+en VS Code mientras ejecutan en el browser.
 
-<p class="tip">Note: This recipe covers Chrome and Firefox. If you know how to setup VS Code debugging with other browsers, please consider sharing your insights (see bottom of the page).</p>
+<p class="tip">
+  Nota: Esta receta cubre Chrome and Firefox. Si tu sabes como
+  configurar la depuración en VS Code con otros navegadores, por favor
+  considera compartir tus percepciones (ver al final de la página).
+</p>
 
-## Prerequisites
+## Prerequisitos
 
-Make sure you have VS Code and the browser of your choice installed, and the latest version of the corresponding Debugger extension installed and enabled:
+Asegúrate de tener VS Code y el navegador de tu preferencia instalado,
```

Comment:
It's perfect, just change it to use the formal language type.
For example: 

- use "Asegúrese" instead of "Asegúrate"
- use "Antes de que pueda" instead of "Antes de que puedas"

Reply:
@sxcamacho I applied those changes and also changed other parts of the code to use 3rd person as well.

Thanks for you great feedback.